uvloop/cbhandles.pyx

```diff
@@ -76,8 +76,8 @@ cdef class Handle:
                     self.arg1, self.arg2, self.arg3, self.arg4)
 
             else:
-                raise RuntimeError('invalid Handle.cb_type: {}'.format(
-                    cb_type))
+                raise RuntimeError('invalid Handle.cb_type: {} {!r}/{!r}'.format(
+                    cb_type, self.args1, self.args2))
```

Comment:
I don't get this one - why logging `args1` and `args2`? (Also I think it's a typo, should be `arg1`?)

Reply:
can be removed, added when doing some debugging